Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.cpp

```diff
@@ -2,8 +2,20 @@
 
 #include "GDKTestGymsFunctionalTests.h"
 #include "LogGymsSpatialFunctionalTest.h"
-#include "Modules/ModuleManager.h"
+#include "TestMapGeneration.h"
 
 DEFINE_LOG_CATEGORY(LogGymsSpatialFunctionalTest);
 
-IMPLEMENT_PRIMARY_GAME_MODULE( FDefaultGameModuleImpl, GDKTestGymsFunctionalTests, "GDKTestGymsFunctionalTests" );
+IMPLEMENT_GAME_MODULE(FGDKTestGymsFunctionalTestsModule, GDKTestGymsFunctionalTests)
```

Comment:
hmmm, will trust you on this one: unfortunately I can never remember what macros, defines, etc. modules need in Unreal